Sources/CrossCompilationDestinationsTool/Configuration/ConfigureDestination.swift

```diff
@@ -12,8 +12,8 @@
 
 import ArgumentParser
 
-struct ConfigureDestination: ParsableCommand {
-    static let configuration = CommandConfiguration(
+public struct ConfigureDestination: ParsableCommand {
```

Comment:
curious why these became public?

Reply:
The reason is that `SwiftDestinationTool` referencing this type had to be moved to a different target so that it could gain the `@main` attribute.